omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,256 @@
+            
+                        if(getValue('current-regimen.value')==""){
+                            jq().toastmessage('showErrorToast', "Current Regimen is required");
+                        }
+
+                        if(getValue('parent-health-facility.value')==""){
+                            jq().toastmessage('showErrorToast', "Parent Health Facility is required");
+                        }
+
+                        if(getValue('serial-number.value')==""){
+                            jq().toastmessage('showErrorToast', "Serial Number is required");
+                        }
+
+                        if(getValue('pt-clinic-number.value')==""){
+                            jq().toastmessage('showErrorToast', "PT Clinic Number is required");
+                        }
+
+                        if(getValue('tb-status.value')==""){
+                            jq().toastmessage('showErrorToast', "The TB Status is required");
+                        }
+
+                        if(getValue('no-of-art-pills.value')==""){
+                            jq().toastmessage('showErrorToast', "The Art Refill is required");
+                        }
+
+                        if (submitValStage.indexOf(false) != -1) {
+                            submitVal = false;
+                            jq().toastmessage('showErrorToast', "There are errors during the validation of the data on the form. Please correct them and submit again");
+                        }
+
+                        return submitVal;
+                    });
+                    
+                   
+
+                });
+    
+            }
+    
+        </script>
+
+    <div>
+        <div class="row">
+            <div class="col-md-6">
+                <h3>Walk-in ART Service</h3>
+            </div>
+            <div class="col-md-5"></div>
+            <div class="col-md-1">
+                <submit class="confirm mb-2" submitLabel="Save" submitStyle="customSubmit" />
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col-md-12">
+                <div class="card">
+                    <div class="card-body">
+
+                        <div class="card">
+                            <div class="card-header">
+                                    Patient History
+                                </div>
+
+                            <div class="card-body">
+                                <div class="row">
+                                    <div class="col-md-3">
+                                        <label>
+                                            <span class="required">*</span>
+                                            Reason for Visit
+                                        </label>
+                                        <obs id="reason-for-visit" conceptId="160288" answerConceptIds="166455, 166456, 165043, 90002" answerLabels="Lockdown, Natural Disaster, Admission, OTHER SPECIFY" />
```

Comment:
@ODORA0 Change "OTHER SPECIFY" to "Other"

Reply:
@ssmusoke Done